src/Components/OATPropertyEditor/Internal/JSONEditor.tsx

```diff
@@ -191,7 +191,8 @@ const JSONEditor: React.FC<JSONEditorProps> = (props) => {
                 options={{
                     minimap: {
                         enabled: false
-                    }
+                    },
+                    readOnly: true
```

Comment:
Why was this added?

Reply:
It's an unrelated ask since we can't fix it before tomorrow.